src/main/java/com/softserve/paymentservice/controller/PaymentController.java

```diff
@@ -0,0 +1,83 @@
+package com.softserve.paymentservice.controller;
+
+import com.softserve.paymentservice.converter.CardDtoToModel;
+import com.softserve.paymentservice.dto.CardDto;
+import com.softserve.paymentservice.dto.PaymentDto;
+import com.softserve.paymentservice.dto.PaymentInfoDto;
+import com.softserve.paymentservice.exception.CardNotFoundException;
+import com.softserve.paymentservice.model.Card;
+import com.softserve.paymentservice.model.Invoice;
+import com.softserve.paymentservice.service.AmountCalculator;
+import com.softserve.paymentservice.service.CardService;
+import com.softserve.paymentservice.service.InvoiceService;
+import com.softserve.paymentservice.service.PaymentService;
+import com.stripe.exception.StripeException;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("api/payment")
+public class PaymentController {
+
+    private final AmountCalculator amountCalculator;
+    private final CardService cardService;
+    private final InvoiceService invoiceService;
+    private final PaymentService paymentService;
+
+    @PostMapping("/card/add")
+    ResponseEntity<Card> addCard(@RequestBody CardDto cardDto) {
```

Comment:
Spring will have to make these methods accessible for reflection, to prevent that it's better to make them public. Also, this breaks suggestions from IDE